alcs-api/src/comment/comment.controller.ts

```diff
@@ -49,12 +49,14 @@ export class CommentController {
     @Body() comment: CreateCommentDto,
     @Req() req,
   ): Promise<CommentDto> {
+    const mappedMentions = await this.mapMentions(comment);
     const newComment = await this.commentService.create(
       comment.fileNumber,
       comment.body,
       req.user.entity,
-      await this.mapMentions(comment),
+      mappedMentions,
```

Comment:
Is there any specific reason or just code-style here?

Reply:
I had originally changed this because I added some code here, but when I ended up re-working it and not needing the mappedMentions variable. I can revert this if you'd like.

Reply:
Feel free to leave it as is, I was just curious about the reason